ui/appui-react/src/appui-react/cursor/cursorpopup/CursorPopup.tsx

```diff
@@ -30,7 +31,7 @@ export interface CursorPopupProps extends CommonProps {
   content: React.ReactNode;
   pt: XAndY;
   offset: XAndY;
-  relativePosition: RelativePosition;
+  relativePosition: RelativePosition | Placement;
```

Comment:
Should we just deprecate `relativePosition` instead for better semantics? Placement is usually optional anyways.
```suggestion
  /** @deprecated in 4.10.0. Use `placement` instead. */
  relativePosition: RelativePosition;
  placement?: Placement;
```

Reply:
Done